<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLeftWall</class>
 <widget class="QWidget" name="WLeftWall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Cantina</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf1Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="topBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf3Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf2Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf8Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf7Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf4Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf6Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf5Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floorBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Floor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allObjsBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>All Objects</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
